src/menu/views/settings_editor.c

```diff
@@ -61,6 +83,12 @@ static void set_rumble_enabled_type (menu_t *menu, void *arg) {
 // }
 #endif
 
+static component_context_menu_t set_loading_progress_bar_enabled_context_menu = { .list = {
```

Comment:
This should not be available in the UI settings editor. It was added for a specific use case where users that required autoload and had scalers had issues with the flipping of resolution. The autoload function is now deprecated and due to be removed as we now have fast reboot.